src/main/resources/init.sql

```diff
@@ -5,24 +5,30 @@ CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 CREATE EXTENSION IF NOT EXISTS citext;
 
 CREATE TABLE event(
-	event_id uuid DEFAULT uuid_generate_v4 () PRIMARY KEY NOT NULL,
+	event_id uuid PRIMARY KEY NOT NULL,
 	name VARCHAR(255),
 	description CITEXT,
 	user_slots INTEGER,
     interval DECIMAL,
-    start_date DATE
+    start_date DATE,
+    duration DECIMAL
 );
 
-CREATE TABLE users(
+CREATE TABLE user(
 	mail VARCHAR(80) PRIMARY KEY NOT NULL,
 	name VARCHAR(50)
 );
 
 CREATE TABLE registered(
```

Comment:
We should verify if we need JPA annotation or not (performance gain ? structure logical ?)